spec/lib/vmdb/loggers_spec.rb

```diff
@@ -84,39 +83,47 @@ def in_container_env(example)
       context "#level" do
         it "defaults the loggers to their default levels" do
           expect(subject.level).to        eq(Logger::INFO)
-          expect($container_log.level).to eq(Logger::DEBUG) if $container_log # $container_log is always DEBUG
+          expect(container_log.level).to eq(Logger::DEBUG) if container_log # container_log is always DEBUG
```

Comment:
not for this pr but for future consideration, does it make sense for us to not need to log so much? From a performance statement and also a difficulty in spelunking statement

Reply:
yes, I have a separate PR that undoes this decision - we just haven't released it yet: https://github.com/ManageIQ/manageiq-loggers/pull/35